examples/CalcQt/features/step_definitions/QtTestCalculatorQtSteps.cpp

```diff
@@ -0,0 +1,72 @@
+#include <QTest>
+#include <cucumber-cpp/autodetect.hpp>
+#include <cstdlib>
+#include <QApplication>
+#include "CalculatorWidget.h"
+
+static int argc = 0;
+static QApplication app(argc, 0);
+static int milliseconds = -1;
+
+int millisecondsToWait() {
+    if (milliseconds < 0)
+    {
+        char* envVariable = getenv("CALCQT_STEP_DELAY");
+        milliseconds = (0 != envVariable) ? atoi(envVariable) : 0;
+    }
+    return milliseconds;
+}
+
+std::istream& operator>> (std::istream& in, QString& val) { std::string s; in >> s; val = s.c_str(); return in; }
+std::ostream& operator<< (std::ostream& out, const QString& val) { out << val.toLatin1().data(); return out; }
```

Comment:
Not even Windows has Latin-1 as default always. It's dependent on the locale, e.g. when set to Russian it has some Cyrillic code page.

Anyway, I suggest you try [toLocal8Bit()](https://doc.qt.io/qt-4.8/qstring.html#toLocal8Bit) instead. It uses whatever the locale says to use as encoding.